src/app/stylesheets/custom/question.less

```diff
@@ -0,0 +1,28 @@
+.ribbon-question {
+  margin-top: 15px;
+  background-color: @bg-on-bg-inverse-color;
```

Comment:
We try not to use directly variables in theme.less (eg `@bg-on-bg-inverse-color`) in less files. It is preferable to create a new variable in variables.less (that you will use here).

In variables.less, add:
```
@ribbon-question-bg-color: @bg-on-bg-inverse-color;
```

This gives us the possibility to tweak the colors as much as possible.

Please take this comment in account everywhere there is a customisable color or value.